recipes/Flask-Fixtures/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "Flask-Fixtures" %}
+{% set version = "0.3.8" %}
+{% set sha256 = "2b8191f995896c0dee384b18b0ee236ab636b921856636c9ff1429c1ad10cb8b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
I believe that this recipe can be noarch. Also, try Grayskull to create your recipes, it can take care of a lot of details.

Reply:
@ocefpaf i installed grayskull using pip. But when i run the `grayskull pypi Flask-Fixtures` i get the below error
ImportError: DLL load failed: The specified module could not be found.

Any help to resolve this error would be appreciated :)

Reply:
There is an online version here which might be easier if you just want to do one recipe: https://www.marcelotrevisani.com/grayskull

Reply:
No luck here too ..
![gray_skull_error](https://user-images.githubusercontent.com/42544705/111311562-60a28200-8684-11eb-8aee-c2ba9f9bafda.png)

Reply:
I'll check via the CLI later but pinging @marcelotrevisani here for awareness.

Reply:
Fixed it
https://github.com/conda-incubator/grayskull/issues/217

Reply:
What is the command that you are using to install the package? 
Can you try to install Flask-minify first?

Reply:
I am trying to use online version - [https://www.marcelotrevisani.com/grayskull](https://www.marcelotrevisani.com/grayskull)